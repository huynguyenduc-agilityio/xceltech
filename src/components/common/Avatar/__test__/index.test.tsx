import { render } from '@testing-library/react';

import { Avatar } from '..';

describe('Avatar component', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <Avatar src="https://github.com/shadcn.png" size={100} />,
    );

    expect(container).toMatchSnapshot();
  });
});
