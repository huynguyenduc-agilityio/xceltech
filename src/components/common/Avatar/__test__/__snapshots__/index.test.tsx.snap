// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component should match snapshot 1`] = `
<div>
  <span
    class="relative flex h-10 w-10 shrink-0 overflow-hidden rounded-full"
    style="width: 100px; height: 100px;"
  >
    <span
      class="flex h-full w-full items-center justify-center rounded-full bg-muted"
    />
  </span>
</div>
`;
